modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/syncml/impl/SyncmlServiceImpl.java

```diff
@@ -323,11 +323,9 @@ private boolean enrollDevice(Document request) throws WindowsDeviceEnrolmentExce
         String deviceName;
         int msgID;
         SyncmlDocument syncmlDocument;
-        SyncmlHeader syncmlHeader;
 
         try {
             syncmlDocument = SyncmlParser.parseSyncmlPayload(request);
-            syncmlHeader = syncmlDocument.getHeader();
             msgID = syncmlDocument.getHeader().getMsgID();
             if (msgID == PluginConstants.SyncML.SYNCML_FIRST_MESSAGE_ID) {
                 Replace replace = syncmlDocument.getBody().getReplace();
```

Comment:
It's inappropriate to use a verb like Replace for a class name, should be some appropriate noun